react-design/my-app/src/RotatingBanner.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react';
+import Banner from './Banner';
+import NextButton from './NextButton';
+import Indicators from './Indicators';
+import PrevButton from './PrevButton';
+export default function RoratingBanner({ itemList }) {
+  const [index, setIndex] = useState(0);
+  const handleClick = (event) => {
+    if (event.target.name === 'Prev') {
```

Comment:
Figure out how to do this logic without examining the `name` prop. Doing this by examining the `name` prop is frowned upon in React code. You can do it with 3 different functions, most (or all) of which can be inline arrow functions.